src/main/java/at/hannibal2/skyhanni/features/gui/customscoreboard/ScoreboardElements.kt

```diff
@@ -804,16 +804,18 @@ private fun getFooterDisplayPair() = listOf(
 private fun getExtraDisplayPair(): List<ScoreboardElementType> {
     if (unknownLines.isEmpty()) return listOf("<hidden>" to HorizontalAlignment.LEFT)
 
-    if (amountOfUnknownLines != unknownLines.size && config.unknownLinesWarning) {
+    val size = unknownLines.size
+    if (amountOfUnknownLines != size && config.unknownLinesWarning) {
+        val message = "CustomScoreboard detected $size unknown line${if (size > 1) "s" else ""}"
```

Comment:
Don't we have some sort of utils to pluralise stuff

Reply:
already got fixed in another pr